ShortcutsSwiftXcode.playground/Sources/Support.swift

```diff
@@ -0,0 +1,17 @@
+import ShortcutsSwift
+import Foundation
+import PlaygroundSupport
+
+public func saveShortcut(_ shortcut: PropertyList, name: String) {
+    let data = exportShortcut(shortcut)
+    func save(ext: String) {
+        let fileURL = URL(fileURLWithPath: "")
+            .appendingPathComponent(name)
+            .appendingPathExtension(ext)
+        try! data.write(to: fileURL)
+        print("open \(fileURL.deletingLastPathComponent().absoluteString.dropFirst(7))")
+    }
+    save(ext: "shortcut")
+    // TESTING
+    save(ext: "plist")
```

Comment:
I like to save a plist version of the shortcut to be able to open it and compare it with the original.